O365/connection.py

```diff
@@ -534,7 +535,7 @@ def get_session(self, *, state=None,
 
         if load_token:
             # gets a fresh token from the store
-            token = self.token_backend.get_token()
+            token = self.token_backend.load_token()
```

Comment:
I don't understand this. If you don't use `get_token` then the token backend 'token' property will remain unset. Therefore the state is inconsistent between the session and the token backend. I think this is fixed with the change I'll introduce later (the one I commented on in the Issue).

Reply:
unintentional - i was debugging & inspecting the behavior of some of your changes and forgot to revert this back to `get_token`